tools/replay-archive/src/cli.rs

```diff
@@ -439,11 +437,8 @@ impl ReplayController {
         {
             bail!("Failed to validate chunk proofs");
         }
-        // FIXME: this should be using Chain::validate_chunk_transactions instead of doing its own
-        // thing?
-        if !validate_transactions_order(chunk.transactions()) {
-            bail!("Failed to validate transactions order in the chunk");
-        }
+        // FIXME: should we be calling Chain::validate_chunk_transactions here as well?
```

Comment:
Need to resolve before merging

Reply:
not sure what you mean.  I just reworded the existing fixme.  I would say that this is an existing issue and not part of this PR.

Reply:
Got it!